session/bootstrap.go

```diff
@@ -604,6 +604,31 @@ const (
        KEY (create_time),
        KEY (create_user));`
 
+	// CreateRunawayTable stores the query which is identified as runaway or quarantined because of in watch list.
+	CreateRunawayTable = `CREATE TABLE IF NOT EXISTS mysql.runaway_queries (
```

Comment:
```suggestion
	CreateRunawayTable = `CREATE TABLE IF NOT EXISTS mysql.tidb_runaway_queries (
```

I suggest using `tidb_xxx`  as the name Indicates tidb specific table.

Reply:
+1.